lib/openhab/core/items/generic_item.rb

```diff
@@ -207,6 +207,14 @@ def format_type(type)
           type.to_s
         end
 
+        # @method set_time_series(time_series)
```

Comment:
I was torn about that, since time_series= seems like such a casual assignment, vs some sort of "update" process. But OK, I'll change it to time_series=.